contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -148,6 +148,19 @@ contract OptimismPortal {
         emit TransactionDeposited(from, _to, msg.value, _value, _gasLimit, _isCreation, _data);
     }
 
+    /**
+     * @notice Determine if an L2 Output is finalized
+     * @param _l2Timestamp The timestamp of the L2 block
+     */
+
+    function isOutputFinalized(uint256 _l2Timestamp) external view returns (bool) {
+        L2OutputOracle.OutputProposal memory proposal = L2_ORACLE.getL2Output(_l2Timestamp);
+        if (proposal.outputRoot == bytes32(uint256(0))) {
+            return false;
+        }
+        return block.timestamp > proposal.timestamp + FINALIZATION_PERIOD;
```

Comment:
Can alternatively combine the above IF statement into this to make a single unified assertion. I'm ok either way.

Reply:
+1, but also fine with it as is.